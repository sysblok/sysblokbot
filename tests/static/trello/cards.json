[
  {
    "id": "card_1",
    "checkItemStates": null,
    "closed": false,
    "dateLastActivity": "2020-04-11T14:30:36.649Z",
    "desc": "Даня сам напишет.\nhttps://terraoblita.com/ru\nСм. https://terraoblita.com/ru/about",
    "descData": null,
    "dueReminder": 1440,
    "idBoard": "board_1",
    "idList": "list_1",
    "idMembersVoted": [],
    "idShort": 56,
    "idAttachmentCover": null,
    "idLabels": [
      "5e91d48a9e9d947b95fb0fd7"
    ],
    "manualCoverAttachment": false,
    "name": "Open Memory Map",
    "pos": 4565421785088,
    "shortLink": "PyXDoStd",
    "isTemplate": false,
    "badges": {
      "attachmentsByType": {
        "trello": {
          "board": 0,
          "card": 0
        }
      },
      "location": false,
      "votes": 0,
      "viewingMemberVoted": false,
      "subscribed": false,
      "fogbugz": "",
      "checkItems": 0,
      "checkItemsChecked": 0,
      "checkItemsEarliestDue": null,
      "comments": 0,
      "attachments": 0,
      "description": true,
      "due": "2020-06-18T09:00:00.000Z",
      "dueComplete": false
    },
    "dueComplete": false,
    "due": "2020-06-18T09:00:00.000Z",
    "idChecklists": [],
    "idMembers": [],
    "labels": [
      {
        "id": "5e91d48a9e9d947b95fb0fd7",
        "idBoard": "5e91d48a9e9d947b95fb0df7",
        "name": "История",
        "color": "yellow"
      }
    ],
    "shortUrl": "https://trello.com/c/card_1",
    "subscribed": false,
    "url": "https://trello.com/c/PyXDoStd/56-terra-oblita-open-memory-map-%D0%BA%D0%B0%D1%80%D1%82%D0%B0-%D0%BF%D0%B0%D0%BC%D1%8F%D1%82%D0%B8-%D0%B7%D0%B0%D0%B1%D1%8B%D1%82%D1%8B%D1%85-%D0%B6%D0%B5%D1%80%D1%82%D0%B2-%D0%BD%D0%B0%D1%86%D0%B8%D0%B7%D0%BC%D0%B0",
    "cover": {
      "idAttachment": null,
      "color": null,
      "idUploadedBackground": null,
      "size": "normal",
      "brightness": "light"
    }
  },
  {
    "id": "card_2",
    "checkItemStates": null,
    "closed": false,
    "dateLastActivity": "2020-04-11T14:30:36.194Z",
    "desc": "",
    "descData": null,
    "dueReminder": null,
    "idBoard": "board_1",
    "idList": "list_3",
    "idMembersVoted": [],
    "idShort": 96,
    "idAttachmentCover": null,
    "idLabels": [],
    "manualCoverAttachment": false,
    "name": "тестовая карточка",
    "pos": 4565421850624,
    "shortLink": "UnqBPZ8h",
    "isTemplate": false,
    "badges": {
      "attachmentsByType": {
        "trello": {
          "board": 0,
          "card": 0
        }
      },
      "location": false,
      "votes": 0,
      "viewingMemberVoted": false,
      "subscribed": false,
      "fogbugz": "",
      "checkItems": 0,
      "checkItemsChecked": 0,
      "checkItemsEarliestDue": null,
      "comments": 0,
      "attachments": 0,
      "description": false,
      "due": null,
      "dueComplete": false
    },
    "dueComplete": false,
    "due": null,
    "idChecklists": [],
    "idMembers": ["member_1"],
    "labels": [],
    "shortUrl": "https://trello.com/c/card_2",
    "subscribed": false,
    "url": "https://trello.com/c/UnqBPZ8h/96-%D1%82%D0%B5%D1%81%D1%82%D0%BE%D0%B2%D0%B0%D1%8F-%D0%BA%D0%B0%D1%80%D1%82%D0%BE%D1%87%D0%BA%D0%B0",
    "cover": {
      "idAttachment": null,
      "color": null,
      "idUploadedBackground": null,
      "size": "normal",
      "brightness": "light"
    }
  }
]
