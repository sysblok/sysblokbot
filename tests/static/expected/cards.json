[
    {
      "id": "card_1",
      "name": "Open Memory Map",
      "due": "2020-06-18T09:00:00.000000Z",
      "members": [],
      "labels": [
        {
          "id": "5e91d48a9e9d947b95fb0fd7",
          "name": "История",
          "color": "yellow"
        }
      ],
      "url": "https://trello.com/c/card_1",
      "list": {
        "id": "list_1",
        "name": "Готовая тема - автор, бери!",
        "idBoard": "board_id"
      }
    },
    {
      "id": "card_2",
      "due": null,
      "name": "тестовая карточка",
      "members": [
        {
          "id": "member_1",
          "username": "paulinmatavina",
          "fullName": "Paulin Matavina"
        }
      ],
      "list": {
        "id": "list_2",
        "name": "Уже пишу (не забудь указать дату и автора!)",
        "idBoard": "board_id"
      },
      "url": "https://trello.com/c/card_2",
      "labels": []
    }
  ]
